platforms/android/com/crittercism/plugin/CDVCrittercism.java

```diff
@@ -57,16 +57,22 @@ public void initialize(CordovaInterface cordova, CordovaWebView webView) {
         Context context = this.cordova.getActivity();
         packageName = context.getPackageName();
         Resources resources = context.getResources();
-        String crAppID = context.getString(resources.getIdentifier(
-                APPLICATION_ID, STRING, packageName));
-        Crittercism.initialize(context, crAppID);
+        int identifier = resources.getIdentifier(APPLICATION_ID, STRING, packageName);
```

Comment:
For customers that were previously on the old system that switch to the new system, I assume their app ID will end up hanging around in their resources.  Would this result in the app getting initialized twice?  The native code protects against this, but they may get a warning message.